### PR TITLE
add in properties forecast key and add ru language

### PR DESCRIPTION
**Pull request by katorzhin**
add forecast api key in properties and add ru language

### CODE REVIEW

File: src/main/resources/messages_ru.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-timezone=Timezone: {0}
-time=Time is: {0}
-summary=Summary: {0}
-icon=Icon:{0}
\ No newline at end of file
+timezone=��������: {0}
+time=�����: {0}
+summary=������: {0}
+icon=������:{0}
```
Comment (GetFire) [inline src/main/resources/messages_ru.properties]: @katorzhin , please check file coding

File: src/main/java/myproject/likelihoodofnaturaldisaster/controller/GeoController.java [hunk 2]
```diff
@@ -21,9 +22,10 @@
     @Autowired
     private AlexeyDtoMapper mapper;
 
-
     private static final String GEO_PAGE = "geoPage";
-    public static final String API_KEY = "321454184d08ad9ad1910cf541bab553";
+
+    @Value("${321454184d08ad9ad1910cf541bab553}")
```
Comment (GetFire) [inline src/main/java/myproject/likelihoodofnaturaldisaster/controller/GeoController.java]: @katorzhin , WTF???? 
you described this value in app.properties but use the key itself??
BTW I asked you to change only message.properties. Why this file appear in your PR?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,5 +2,7 @@ spring.mvc.view.prefix=/WEB-INF/view/
 spring.mvc.view.suffix=.jsp
 server.port=${Port:8080}
 
+forecast.api.key=321454184d08ad9ad1910cf541bab553
```
Comment (GetFire) [inline src/main/resources/application.properties:5]: @katorzhin , why this file appears in this PR?

**Review by GetFire — CHANGES_REQUESTED** [review]

File: src/main/resources/messages_ru.properties (hunk 1 above)
Comment (katorzhin) [reply src/main/resources/messages_ru.properties]: @GetFire checked

**Review by GetFire — APPROVED** [review]

File: src/main/java/myproject/likelihoodofnaturaldisaster/controller/GeoController.java (hunk 2 above)
Comment (katorzhin) [reply src/main/java/myproject/likelihoodofnaturaldisaster/controller/GeoController.java]: @GetFire  you asked me to do it earlier, so I did like you told me

**Pull request merged**